### DISCUSSION TITLE
Allow notification responses for previous day

### DISCUSSION THREAD

**Discussion by tunbridgep** [iSoron/uhabits#1205] [Feature Requests]
I like the fact that you can fill out a habit from the notification. "Did you read today?" with "yes", "No" etc. It's a good feature and means I spend less time having to manually go into the app to fill things out.

However, I have several habits where the notification doesn't really make sense on the day of doing the task. For example, I read every night right before bed, and usually put my book down and immediately fall asleep when I feel tired. The best time for me to fill out a notification is the next day at 9am when I wake up.

There are currently a few workarounds for this. I can set the notification for 11pm and just not touch my phone until morning, I can just go into the app and fill it out manually, or I can restructure the habit to be "did you read last night?". However I don't like these approaches as I either get bombarded with notifications when I'm trying to sleep or the habit is offset by 1 day, and I feel it would be neater if I could set the notification to fill out for the previous day, so I can still track my habit like a normal daily activity.

So at 9am the next morning I would wake up to a notification "Did you read last night?" "Yes" "No". Pressing Yes or No will tick the box for the previous day, not the current day, so my habit isn't offset by a day. I feel like it's one of those nice little features that would make things easier to use.

Also, I noticed there's no "Skip" option in the notification when skip days are enabled. Is this intentional?

**Comment by Ackleberry** [comment]
+1 came here to request this feature. I'm envisioning when you edit/create a habit there is a "Reminder applies to previous day" checkbox. Then I can have a reminder set for 9am, which asks me about my habits that I did the previous day. 

Currently I'm just living with everything being off by 1 day.
